doc/user/methods/gradient_boosting.md

```diff
@@ -0,0 +1,227 @@
+## `GradBoosting`
+
+The `GradBoosting` class implements a Gradient Boosting model that supports
+numerical and categorical features. The class offers several template 
```

Comment:
Does this actually support categorical features?  I don't think it does currently.